packages/core/src/Utils/Datadog/index.ts

```diff
@@ -41,5 +41,6 @@ datadogRum.init({
     version: dataDogVersion,
     trackFrustrations: true,
     enableExperimentalFeatures: ['clickmap'],
+    excludedActivityUrls: ['TELEGRAM_ACCESS_TOKEN'],
```

Comment:
just a question, shouldn’t it be a regex url like `“^https://api.telegram.org/.*”`, because this is what shows as resource url

Reply:
Yes I think that makes sense.I will  try that